src/resources/i18n/translations/en.json

```diff
@@ -7,17 +7,17 @@
 	"ARIA HINT - Error something went wrong with playing this item": "Could not play this.",
 	"ARIA HINT - Hide podcast settings": "Hide podcast settings",
 	"ARIA HINT - Max playback speed": "Set max playback speed for your player",
-	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse premium.",
-	"ARIA HINT - Membership - Create playlists": "Create playlists with Podverse premium.",
-	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data with Podverse premium.",
+	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse Premium.",
+	"ARIA HINT - Membership - Create playlists": "Create playlists with Podverse Premium.",
+	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data with Podverse Premium.",
 	"ARIA HINT - Membership - Download episodes": "Download episodes.",
 	"ARIA HINT - Membership - Drag and drop queue": "Player queue.",
 	"ARIA HINT - Membership - Sleep timer": "Sleep timer.",
 	"ARIA HINT - Membership - Subscribe to podcasts": "Subscribe to podcasts.",
-	"ARIA HINT - Membership - Support free and open source software": "Support copylefted libre software 😊",
-	"ARIA HINT - Membership - Sync your queue on all devices": "Sync your queue to all devices with Podverse premium.",
-	"ARIA HINT - Membership - Sync your subscriptions on all devices": "Sync your subscriptions to all devices with Podverse premium.",
-	"ARIA HINT - Membership features header": "Features the app presented as gratis, and premium features.",
+	"ARIA HINT - Membership - Support free and open source software": "Support free and open source software 😊",
```

Comment:
```suggestion
	"ARIA HINT - Membership - Support free and open source software": "Copylefted libre software 😊",
```

>no Americans could tell you what "libre software" is.

Americans both know what the root word liberty and what it means.

>I'm somewhat aware of the semantic battles between "open source vs libre" software, but for now, we're describinig us as an open source app rather than a libre app. I know it's not the most accurate, but we think open source will make more sense to American users.

Unless you are specific about the OSI owning the term and using it to describe a set selection of licenses, it means the same thing, but one purposefully tries to not focus on freedom, in place of business merchantability.

>I try to add "free and open source" to indicate we make more than just open source (FOSS), but that gets confusing too because we offer a paid premium membership option...so we're opting for open source for now.

You don't though. "free and open source" also means just "libre", but now it is super confusing since one seems to qualify the other. Moreover if it wasn't to mean free as in gratis/free of charge, convention is to use just one word to describe one thing. So more is implied, but nothing more is communicated.

>Maybe in other countries Libre has a more well-understood meaning and we can use clearer words.

Doesn't help when carrying over this duality in English over to langauges where it doesn't belong.
Translations will say +~"and gratis" and that means selling the concept of freedom short.

The projects using "copylefted libre" get more donations in my experience. Despite its frequency it actually communicates something other than "this is cool as in freeware", which is what most people sadly will tell you about "open source" if they know at all.

Reply:
I appreciate the distinctions you're drawing, but in this specific line we want to go with "free and open source" instead of "copylefted libre" 

We use the term copyleft already in some places. I'd like to use a combination of  "free and open source", "FOSS", "gratis", "copyleft", and "copylefted libre" software. All of these terms are worth more people understanding. I'll make some commit suggestions.

Reply:
Ok, go ahead with `Copylefted libre software 😊` but can you change it to `Support copylefted libre software 😊`?

Reply:
@comradekingu I forgot this was a PR I opened when I left these comments 🤦‍♂️ let me know if you have any objections but I'll make the changes myself.

Reply:
Nevermind, this is the first time I've worked with a lot of suggestions in a PR like this. Slowly figuring it out 😬

Reply:
If you put in all my changes (or as many as possible) by adding them all and then committing it is all good.